scenarios/kubernetes_build.py

```diff
@@ -34,8 +34,6 @@ def check(*cmd):
 def main(args):
     """Build kubernetes."""
     env = {}
-    if args.unstable:
-        env['DEB_CHANNEL'] = 'unstable'
```

Comment:
I've searched Google, and the only place I've seen this variable turn up is the test-infra repo.
Furthermore, [`release/debian/jenkins.sh`](https://github.com/kubernetes/release/blob/master/debian/jenkins.sh) basically just runs a docker container without passing in the environment, so I really don't think this variable has any effect.